### DISCUSSION TITLE
Export collections from osu!lazer

### DISCUSSION THREAD

**Discussion by Modsnotworking** [ppy/osu#14658] [Q&A] [answered]
I want to export my **3 different collections** _simutaneously_ from osu!lazer to a **different location of my pc**.
How am I suppose to export my collections from osu!lazer??

**Comment by peppy** [comment, marked as answer]
There's no export functionality yet, but the `collections.db` file is both portable and cross-compatible with stable for the time being, so you can actually just copy it across to the other PC's data folder.

**Reply by Modsnotworking** [reply]
> There's no export functionality yet, but the `collections.db` file is both portable and cross-compatible with stable for the time being, so you can actually just copy it across to the other PC's data folder.

hmm can I copy and paste it to osu!(stable) folder, and after I do that will the collections appear?

**Reply by peppy** [reply]
Yes, that should work in theory.

**Reply by donglee0612** [reply]
> Yes, that should work in theory.

Where would the collection.db in lazer? do I just copy and paste it into the main osu folder?

**Reply by Rekunan** [reply]
> > Yes, that should work in theory.
> 
> Where would the collection.db in lazer? do I just copy and paste it into the main osu folder?

I think they mean to copy to a stable install and then opening lazer to import it from stable.

**Reply by peppy** [reply]
`collections.db` no longer exists, as we have migrated collections to realm.

**Reply by ghost** [reply]
I have written [a simple tool](https://github.com/ookiineko/CollectionDowngrader) to downgrade the collection data from the lazer realm database back to the old collection.db format (so you can paste the downgraded collection.db into stable), and you need to use [BeatmapExporter](https://github.com/kabiiQ/BeatmapExporter) to import all your beatmaps from lazer first. 
**Note: Importing data from lazer to stable is not officially supported by osu. Use this tool at your own risk and make sure to backup your data before doing so.**

**Reply by huyenden** [reply]
Can you update CollectionDowngrader? Because the client.realm I am currently using has an error recording Date Ranked, I really need to transfer the collection back to stable and then replace it with a new realm.

**Reply by ghost** [reply]
> Can you update CollectionDowngrader? Because the client.realm I am currently using has an error recording Date Ranked, I really need to transfer the collection back to stable and then replace it with a new realm.

sorry! i forgot about archived repo cannot receive new issues.

not sure about what you are trying to do but yes, i can try to update. and just now i have created a new pre-release in my repo. i wasn't maintaining it just because personally im no longer using it, so please let me know if it work or not.

also since i have now unarchived the repo, let's move this topic to a corresponding issue under my repo.

**Reply by huyenden** [reply]
!! Thank you very much. I may now have the mind to transfer the collections played from osu lazer from june 2023 to the new realm. Regarding the error, it is because osu!lazer currently cannot read SHA32s beatmaps in older realm so now you have save me

**Reply by huyenden** [reply]
> not sure about what you are trying to do but yes, i can try to update. and just now i have created a new pre-release in my repo. i wasn't maintaining it just because personally im no longer using it, so please let me know if it work or not.

Yep, totally works. Thank you very much
